docker/openidm2/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM forgerock-docker-public.bintray.io/forgerock/openidm:6.0.0
```

Comment:
I guess openidm2 is for testing? It would be good to have only a single openidm docker image.
A default config is OK here.

Let's put this PR through - but you can do another one that merges the two docker images together.